pyroute2/dhcp/hooks.py

```diff
@@ -24,21 +25,54 @@ async def unbound(self, lease: Lease):
 
 class ConfigureIP(Hook):
     async def bound(self, lease: Lease):
-        LOG.info('STUB: add %s to %s', lease.ip, lease.interface)
-        # await ip(
-        #     "addr",
-        #     "replace",
-        #     f"{lease.ip}/{lease.subnet_mask}",
-        #     "dev",
-        #     lease.interface,
-        # )
+        LOG.debug('Adding IP %s to %s', lease.ip, lease.interface)
```

Comment:
I'd remove `IP` (since it's obvious that it is an IP when reading the log), and add the subnet mask. Maybe the log level would be better as INFO since it's pretty much doing the most important thing we expect from a DHCP client.